src/Airport.js

```diff
@@ -0,0 +1,49 @@
+const Plane = require('./Plane');
+
+class Airport {
+
+    planesAtAirport = [];
+
+    constructor(airportCapacity = 10) {
+        this.airportCapacity = airportCapacity;
+    }
+
+    landPlane(plane, weather) {
+        if (weather === `sunny` && plane instanceof Plane) {
+            if (!this.isFull() && !this.isPlaneAtAirport(plane.getID())) {
+                this.planesAtAirport.push(plane)
```

Comment:
**landPlane method:** I had created two IF statements in this method, however unsure if this should have just been 1? My thoughts were on not wasting time running through 1 long IF statement if the conditions weather===sunny and plane instanceof Plane werent met, however on review this could have been 1 IF statement instead of 2.